docs/ui/installation.rst

```diff
@@ -42,7 +42,7 @@ that new code has a consistent style and doesn't suffer from common
 errors. Before submitting a patch, check that your code passes these tests.
 
 * If you haven't already done so, install local dependencies by
-  running as root ``npm install`` from the project root
+  running as root ``npm install -g grunt-cli`` from the project root
```

Comment:
Ok, this is actually still wrong. :)

change this to: 
    install local dependencies by running ``npm install`` from the project root (root is not required). Then run ``npm 
    install -g grunt-cli``as root, so that you can run grunt.

Please modify the instructions for unit tests (just below this one) as well.